gluoncv/model_zoo/siamrpn/siam_net.py

```diff
@@ -0,0 +1,74 @@
+"""SiamRPN network
+Reference:
+Li, Yan, et al. "High Performance Visual Tracking with Siamese Region Proposal Network
+Object tracking, CVPR 2018
+http://openaccess.thecvf.com/content_cvpr_2018/papers/Li_High_Performance_Visual_CVPR_2018_paper.pdf
+"""
+
+# pylint: disable=arguments-differ,unused-argument
+from mxnet.gluon.block import HybridBlock
+from gluoncv.model_zoo.siamrpn.siam_alexnet import alexnetlegacy
+from gluoncv.model_zoo.siamrpn.siam_rpn import DepthwiseRPN
+
+class SiamrpnNet(HybridBlock):
```

Comment:
Maybe change the name `SiamrpnNet` to `SiamRPN`. Because the class name should match what people usually call these models, such as SiamFC, PSPNet, ResNet, etc.